config/locales/en/helpers.yml

```diff
@@ -155,6 +157,8 @@ en:
         other_names: This includes aliases, nicknames or other names your client may be known as.
         date_of_birth: For example, 27 3 2007
       steps_client_has_nino_form:
+        has_nino: It’s on their National Insurance card, benefit letter, payslip or P60. For example, ‘QQ 12 34 56 C’.
+        new_nino_hint: For example, QQ 12 34 56 C
```

Comment:
Once the means journey code is in this can be refactored to have the label `nino` but for now that will be break the current form. It's the same with the label in the steps.yml file. This can be addressed in [CRIMAPP-660](https://dsdmoj.atlassian.net/browse/CRIMAPP-660)